### ISSUE TITLE
German name "fahrplan" makes non-german people think this is not for them

### ISSUE THREAD

**Issue by mzanetti** [smurfy/fahrplan#184]
In the Ubuntu store I see an increasing amount of train schedule apps popping up for various countries. I believe this is because the name "fahrplan" suggests this app would only work in germany. Besides the obvious solution here, which is to change the name but might not be wanted at this point any more, I could think of some other solutions to tackle this. For example we'd need to a) be really clear with the description of the app and b) make sure the keywords in the various stores reflect all supported backends, but also include things like "train", "bus", "journey" and similar things in all the languages of the countries where the app works.

I think it would make sense to have a unified set of descriptions, keywoards and other presentational material for all the app stores the app is published. At the moment I can't find such a guideline/template.

**Comment by mzanetti** [comment]
for the record, this is what I currently use in the Ubuntu App store:

Description:
A Journey planner/Railway Time table for public transportation. Currently supported services are:
* bahn.de 
* oebb.at 
* rejseplanen.dk 
* sbb.ch 
* reiseinfo.no / nri 
* vesttrafik.se 
* ptv.vic.gov.au 
* Sydney 
* SF Bay 
* London 
* Ireland 
* Dubai 
* 9292ov.nl 

Keywoards:
train, bus, travel, bahn, oebb, öbb, rejseplanen, sbb, reiseinfo, vesttrafik, ptv, Sydney, SF Bay, London, Ireland, Dubai, 9292ov, Reisplanner, tube, luas, rail

This is not enough as it seems though...

**Comment by leppa** [comment]
Sounds like a good idea. When I added Fahrplan to the BlackBerry World, I've made an updated description which was later adopted by @smurfy for other stores. Currently, a nice description that can be used in the stores is on http://fahrplan.smurfy.de/. In case of BlackBerry World, the description contains "A journey planner and timetable application for a lot of train and public transport lines in Europe, USA and Australia.", **Fahrplan Features**, and **Currently supported Providers** sections. The keywords are (max 10 allowed): fahrplan, timetable, journey, planner, train, bus, bahn, db, sbb, öbb.

However, we can also add a text / Markdown / whatever file to the repo with suggested description, keywords, other info.

**Comment by smurfy** [comment]
As @leppa mentioned, i updated the website with a more detailed list about the supported providers.
It contains, the official website and the country as-well.
I think we should use this detailed list in the descriptions.

About keywords: we should use the provider names (url).

For application name we could use in the store something like: **fahrplan - the public transportation app**

**Comment by leppa** [comment]
By the way, @smurfy, should it be "**f**ahrplan" or "**F**ahrplan"? Because I was always using "Fahrplan", but noticed that you usually use "fahrplan".

**Comment by smurfy** [comment]
I mainly use fahrplan, because i don't like typing uppercase :)
But you are right, we should specify some kind of reference and the actual notation is important

**Comment by bittner** [comment]
@mzanetti The app's name in the Ubuntu Store is still **fahrplan** (lower case). How is it spelled in other app stores?

@smurfy What about **Fahrplan Worldwide** as an app name? Alternatively, **Fahrplan (worldwide)** which may suggest this is the "international version" of the German public transport app, haha.  :wink: 

@mzanetti I agree the app is hard to find. I think I even didn't find it knowing its name, don't know why (though this might be a *different* issue). Could be that I had to switch the device language to German before I was able to locate the app in the Ubuntu Store (on my bq Aquaris E5).

**Comment by bittner** [comment]
Any thoughts on "**Fahrplan Worldwide**"? This may work out in an international context. A tagline, as proposed above, even localized if localized names of the app are supported in the App Stores, could still be added, e.g.

* :gb: Fahrplan Worldwide - Public Transport Timetable
* :fr: Fahrplan Worldwide - horaire transports publics
* :it: Fahrplan Worldwide - orario di trasporto pubblico 
* :es: Fahrplan Worldwide - horarios de los transportes públicos

(But then again, maybe it works out only for us that we use English on a daily basis. A field test can only tell. What do you guys think?)

**Comment by smurfy** [comment]
Maybe adding "worldwide" helps, if someone searchs and finds the app.
i'm not sure how people normally search for an app like this, but my guess is by provider/backend not by the string "public transport"

**Comment by bittner** [comment]
You're right. But this is covered by the keywords, anyway (provided everyone follows your suggestion from above). As soon as the apps found for, e.g., the provider/backend are listed the user would choose the app that seems relevant enough for her/him.

I would guess users that think locally would then choose an app focused on a local implementation if it exists (e.g. Dublin transport), and users that prefer an app that does it all (for anywhere in the world) would be attracted by the "worldwide" claim of Fahrplan. How does that sound?

**Comment by bittner** [comment]
Any decision made here yet?
